misc/contrib/docker/build/Dockerfile

```diff
@@ -0,0 +1,94 @@
+FROM debian:stable-slim
+LABEL maintainer="it@nkey.com.br"
+
+ARG BUILD=prod
+ARG DOCKER_DIR=./misc/contrib/docker
+ARG uwsgi_uid=700
+ARG uwsgi_gid=700
+
+ENV BUILD=$BUILD
+ENV DOCKER_CONTAINER=1
+ENV UWSGI_INI /nsupdate/uwsgi.ini
+ENV DJANGO_SETTINGS_MODULE=local_settings
+ENV DJANGO_SUPERUSER=django
+ENV DJANGO_SUPERPASS=S3cr3t
+ENV DJANGO_EMAIL=django@nsupdate.localdomain
+ENV SERVICE_CONTACT=hostmaster@nsupdate.localdomain
+ENV SECRET_KEY=S3cr3t
+ENV BASEDOMAIN=nsupdate.localdomain
+ENV REGISTRATION_OPEN=False
+
+RUN mkdir /static
+RUN mkdir /upload
+RUN mkdir /var/run/uwsgi
+
+# Install runtime tools
+RUN DEBIAN_FRONTEND=noninteractive apt-get update \
+    && apt-get install -y --no-install-recommends \
+      python3 \
+      python3-setuptools \
+      python3-pip
+
+# Install confd
+RUN apt-get install -y --no-install-recommends wget \
+    && mkdir -p /usr/local/bin \
+    && wget -O /usr/local/bin/confd https://github.com/kelseyhightower/confd/releases/download/v0.16.0/confd-0.16.0-linux-amd64 \
+    && chmod +x /usr/local/bin/confd \
+    && mkdir -p /etc/confd/{conf.d,templates} \
+    && apt-get autoremove -y wget
+
+# Use local version of nsupdate from sources
+COPY ./*.py /nsupdate/
```

Comment:
does that still work now as the stuff is in misc/contrib/docker?

Reply:
This works because the Makefile is run from the project base dir.